frontend/src/components/PetitionBody.tsx

```diff
@@ -0,0 +1,50 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import { PetitionDetail } from "../models/Petition";
+import { PostDetail } from "../models/Post";
+import Main from "./Main";
+import Markdown from "./Markdown";
+import PetitionCard from "./PetitionCard";
+
+interface Props {
+    post: PostDetail;
+    petition: PetitionDetail;
+    fallbackBanner: string;
+}
+
+export default function PetitionBody(props: Props) {
+    const petition = props.petition;
+    const post = props.post;
+
+    const { asPath } = useRouter();
+    const [signThanks, setSignThanks] = useState(<></>);
+
+    useEffect(() => {
+        const hash = asPath.split("#")[1];
+        if (hash && hash.includes("thanks")) {
+            setSignThanks(
+                <div className="mt-3 font-title text-2xl font-bold bg-primary text-white container">
+                    Bedankt voor het ondertekenen! Misschien kun je de petitie
+                    nu delen?
+                </div>
+            );
+        }
+    }, [asPath]);
+
+    return (
```

Comment:
Kijk alsnog of je die haakjes kan fixen. Op het werk hebben we dat ook aan kunnen maken, alhoewel we een andere linter gebruiken